.github/workflows/sonar-project.properties.yml

```
@@ -0,0 +1,221 @@
+          if [[ -n $(git status -s) ]]; then
+            git add kubernetes/*.yaml
+            git commit -m "Update image tags to ${{ github.run_number }} and ensure correct domain [ci skip]"
+            git push
+          else
+            echo "No changes to commit"
+          fi
+
+      - name: Install kubectl
+        run: |
+          sudo curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          sudo chmod +x kubectl 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)